src/main/scala/mesosphere/marathon/core/history/impl/HistoryActor.scala

```diff
@@ -3,27 +3,32 @@ package core.history.impl
 
 import akka.actor.Actor
 import akka.event.EventStream
+import com.typesafe.scalalogging.StrictLogging
 import mesosphere.marathon.core.event._
 import mesosphere.marathon.state.TaskFailure
 import mesosphere.marathon.storage.repository.TaskFailureRepository
 
 // TODO(PODS): Move from Task to Instance
 class HistoryActor(eventBus: EventStream, taskFailureRepository: TaskFailureRepository)
-  extends Actor {
+  extends Actor with StrictLogging {
 
   override def preStart(): Unit = {
     // TODO(cleanup): adjust InstanceChanged to be able to replace using MesosStatusUpdateEvent here (#4792)
     eventBus.subscribe(self, classOf[MesosStatusUpdateEvent])
     eventBus.subscribe(self, classOf[UnhealthyInstanceKillEvent])
     eventBus.subscribe(self, classOf[AppTerminatedEvent])
+
+    logger.info("Scheduler actor ready")
   }
 
   def receive: Receive = {
 
-    case TaskFailure.FromUnhealthyInstanceKillEvent(taskFailure) =>
+    case m @ TaskFailure.FromUnhealthyInstanceKillEvent(taskFailure) =>
       taskFailureRepository.store(taskFailure)
 
-    case TaskFailure.FromMesosStatusUpdateEvent(taskFailure) =>
+    case m @ TaskFailure.FromMesosStatusUpdateEvent(taskFailure) =>
+      println("mesos update")
+      println(m)
```

Comment:
this is just temporary debugging for the loop. If we decide to merge I'll move it to a different branch